src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -173,41 +153,38 @@ Before starting our [guided install](https://one.newrelic.com/nr1-core?state=51f
 
 <InstallFeedback />
 
-## Upgrading our Kubernetes integration [#update]
+### Use guided install [#installer]
 
-Our Kubernetes integration is under active development and we regularly release updates which include bug fixes, new features, and support for newer Kubernetes versions and cloud providers. We strongly recommend all our customers to regularly update the Kubernetes integration to get the best experience.
+We encourage you to use our guided install for servers, VMs, and unprivileged environments. The guided install can provide you either a Helm command with the required values filled, or a plain manifest if you don't want to use Helm. It also offers control over which features are enabled and which data is collected. If you are installing our integration on a managed cloud, please take a look at these [preliminary notes](#cloud-platforms) before proceeding.
```

Comment:
@nbaenam I don't think we need a comma in this since there's no subject in the second half of the sentence: "The guided install can provide you either a Helm command with the required values filled or a plain manifest if you don't want to use Helm.

Maybe we can personalize this more with "you": It also offers you control over which features are enabled and which data is collected.